docs/docs/api-files-gatsby-ssr.md

```diff
@@ -0,0 +1,24 @@
+---
+title: The gatsby-ssr.js API file
+---
+
+The file `gatsby-ssr.js` lets you alter the content of static HTML files as they are being rendered by Gatsby. To use the [Gatby SSR APIs](/docs/ssr-apis/), create a file called `gatsby-ssr.js` in the root of your site. Export any of the APIs you wish to use in this file.
+
+The APIs `wrapPageElement` and `wrapRootElement` exist in both the SSR and [browser APIs](/docs/browser-apis). If you use one of them, consider if you should implement it in both `gatsby-ssr.js` and `gatsby-browser.js` so that SSR-generated pages are the same after being hydrated in the browser.
```

Comment:
```suggestion
The APIs `wrapPageElement` and `wrapRootElement` exist in both the SSR and [browser APIs](/docs/browser-apis). If you use one of them, consider if you should implement it in both `gatsby-ssr.js` and `gatsby-browser.js` so that pages generated through SSR with Node.js are the same after being [hydrated](/docs/glossary#hydration) with browser JavaScript.
```